### PR TITLE
interactive bots: Improve task descriptions.

### PR DESCRIPTION
**Pull request by HogabHerzl**
Just minor corrections and edits

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@HogabHerzl, thanks for the pull request! It looks like you haven't yet signed the Dropbox CLA. Please [sign it here](https://opensource.dropbox.com/cla/).

File: tasks/interactive-bots.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 #GCI Tasks: interactive bots
 
-## Prerequisites
+## Project Essentials
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: Prerequisites its good and common used to it. Please revert the change.

File: tasks/interactive-bots.md [hunk 2]
```diff
@@ -16,18 +16,16 @@
 
 ## Background
 
-Zulip [contrib-bots](https://github.com/zulip/zulip/tree/master/contrib_bots/lib)
-is a boilerplate for creating interactive bots that react to messages sent
+The Zulip [contrib-bots](https://github.com/zulip/zulip/tree/master/contrib_bots/lib)
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: For me the change (and change below) makes no sense

File: tasks/interactive-bots.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 
-## Prerequisites
+## Project Essentials
 
 * A working Zulip development environment. See
   [here](https://github.com/zulip/zulip-gci/blob/master/README.md) for instructions on
   how to set one up.
 
-* You need to know how to create a GitHub pull request. Check out the
+* Knowledge on how to create a GitHub pull request. Check out the
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: I don't see any reason why we should change it - the previous version is absolutely understandable

File: tasks/interactive-bots.md [hunk 4]
```diff
@@ -16,18 +16,16 @@
...
 
-Zulip [contrib-bots](https://github.com/zulip/zulip/tree/master/contrib_bots/lib)
-is a boilerplate for creating interactive bots that react to messages sent
+The Zulip [contrib-bots](https://github.com/zulip/zulip/tree/master/contrib_bots/lib)
+boilerplate is used for creating interactive bots that react to messages sent
 by users.
 
-The interactive bots live in the `contrib-bots/lib` as python files ('.py') that define
-their specific behavior. The `contrib-bots/run.py` file defines common behaviors
+The interactive bots are constituted of python files ('.py') in `contrib-bots/lib` that define
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: The previous version was understandable much more than yours. Please revert it.

File: tasks/interactive-bots.md [hunk 5]
```diff
@@ -82,13 +80,13 @@ Follow the tutorial below to test run the followup bot on your Zulip instance
   a different folder than `~`, you have to point to the right folder, e.g.
   `../../.zuliprc-local`.
 
-* Test manually that the followup bot is working on the local Zulip instance,
+* Test manually that the followup bot is working on the local Zulip instance
 by:
 
-  a. sending a few messages starting with `@followup` and `@follow-up` from
+  A. sending a few messages starting with `@followup` and `@follow-up` from
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: Unnecessary change. Please revert all of "points"

File: tasks/interactive-bots.md [hunk 6]
```diff
@@ -101,19 +99,19 @@ of:
  - any other screenshots you find relevant
 
 Add the screenshots to `interactive-bots/followup/<username>/`. Make sure your
-filenames include no white spaces or underscores, use dashes (`-`) as connectors.
+filenames use dashes (`-`) in place of spaces or underscores.
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: again, for me the previous version was understandable much more

File: tasks/interactive-bots.md [hunk 7]
```diff
@@ -101,19 +99,19 @@ of:
...
 
-* Note down any places you got stuck, problems or errors you ran into while
-doing this setup process. Add your notes as a `notes.md` file to
+* Take note of any places you got stuck and problems or errors you ran into while
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: Note down sounds better

File: tasks/interactive-bots.md [hunk 8]
```diff
@@ -101,19 +99,19 @@ of:
...
+doing this setup process. Add your notes as a `notes.md` file in
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: I think  `to` is appropraite

File: tasks/interactive-bots.md [hunk 9]
```diff
@@ -101,19 +99,19 @@ of:
...
 `interactive-bots/followup/<username>/`.
 
-* Create a commit with the screenshots and notes, with commit message
+* Create a commit with the screenshots and notes, with the commit message
 `interactive bots: Run the followup bot for <username>.`.
 
-* Create a pull request in the `zulip/zulip-gci` repository, with title
-`interactive bots: Run the followup bot for <username>`. Link to your GCI task in comment section.
+* Create a pull request in the `zulip/zulip-gci` repository, with the title
+`interactive bots: Run the followup bot for <username>`. Include a link to your GCI task in comment section.
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: Not sure but maybe `Add` ?

File: tasks/interactive-bots.md [hunk 10]
```diff
@@ -101,19 +99,19 @@ of:
...
 
-*Completion criteria:* Mentors will check that the followup bot was properly
+*Completion criteria:* Your project mentors will check that the followup bot was properly
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: `Mentors` is enough

File: tasks/interactive-bots.md [hunk 11]
```diff
@@ -146,7 +144,7 @@ messages to users.
   "@link" or "@resource".
 
 
-* Edit the comments of `triage_message`:
+* Edit the comments in `triage_message`:
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: I think `of` was correct

File: tasks/interactive-bots.md [hunk 12]
```diff
@@ -182,13 +180,13 @@ messages to users.
 
 
 * Create a links bot in settings for an existing user. Use its credentials in
-`~/.zuliprc-local`. Subscribe the bot to `devel` and `social` streams.
+`~/.zuliprc-local`. Subscribe the bot to the `devel` and `social` streams.
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: the is unnecessary I think

File: tasks/interactive-bots.md [hunk 13]
```diff
@@ -182,13 +180,13 @@ messages to users.
...
 
-* Check that the bot is working as expected:
+* Check that the bot is working as expected. A functional bot should:
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: `It should` would be enough

File: tasks/interactive-bots.md [hunk 14]
```diff
@@ -182,13 +180,13 @@ messages to users.
...
 
-  - it reacts to the `@link` and `@resource` at the beginning of a message
+  - react to the `@link` and `@resource` at the beginning of a message
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: reacts

File: tasks/interactive-bots.md [hunk 15]
```diff
@@ -182,13 +180,13 @@ messages to users.
...
 
-  - it currently posts the message to the followup stream, as the followup bot
+  - currently post the message to the followup stream, as the followup bot
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: posts

File: tasks/interactive-bots.md [hunk 16]
```diff
@@ -201,7 +199,7 @@ messages to users.
   ))
   ```
 
-* Check that the bot is working as expected:
+* Check that the bot meets these requirements:
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: Before version was understandable enough

File: tasks/interactive-bots.md [hunk 17]
```diff
@@ -213,23 +211,24 @@ of:
  - your terminal window with the bot running, including the command you used to
  run the bot and the output with the bot description
  - messages sent to the bot in the `devel` and `social` streams
- - messages sent by the bot in the private messages to the author
+ - messages sent by the bot in the followup stream
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: Seems like a good catch!

File: tasks/interactive-bots.md [hunk 18]
```diff
@@ -213,23 +211,24 @@ of:
...
  - any other screenshots you find relevant
 
- Add the screenshots to `interactive-bots/links/<username>/`. Make sure your
- filenames include no white spaces or underscores, use dashes (`-`) as connectors.
+Add the screenshots to `interactive-bots/followup/<username>/`. Make sure your
+filenames use dashes (`-`) in place of spaces or underscores.
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: Please revert to previous version

File: tasks/interactive-bots.md [hunk 19]
```diff
@@ -213,23 +211,24 @@ of:
...
 
-* Note down any places you got stuck, problems or errors you ran into while
-doing this setup process. Add your notes as a `notes.md` file to
-`interactive-bots/links/<username>/`.
+* Take note of any places you got stuck and problems or errors you ran into while
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: Note down sounds better

File: tasks/interactive-bots.md [hunk 20]
```diff
@@ -213,23 +211,24 @@ of:
...
 
-* Create a commit with the screenshots and notes, with commit message
-`interactive bots: Run the links bot for <username>.`.
+* Create a commit with the screenshots and notes, with the commit message
+`interactive bots: Run the followup bot for <username>.`.
 
-* Create a pull request in the `zulip/zulip-gci` repository, with title
-`interactive bots: Run the links bot for <username>`. Link to your GCI task in comment section.
+* Create a pull request in the `zulip/zulip-gci` repository, with the title
+`interactive bots: Run the followup bot for <username>`. Include a link to your GCI task in comment section.
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: Add?

File: tasks/interactive-bots.md [hunk 21]
```diff
@@ -213,23 +211,24 @@ of:
  - your terminal window with the bot running, including the command you used to
  run the bot and the output with the bot description
  - messages sent to the bot in the `devel` and `social` streams
- - messages sent by the bot in the private messages to the author
+ - messages sent by the bot in the followup stream
  - any other screenshots you find relevant
 
- Add the screenshots to `interactive-bots/links/<username>/`. Make sure your
- filenames include no white spaces or underscores, use dashes (`-`) as connectors.
+Add the screenshots to `interactive-bots/followup/<username>/`. Make sure your
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: links

File: tasks/interactive-bots.md [hunk 22]
```diff
@@ -213,23 +211,24 @@ of:
...
+filenames use dashes (`-`) in place of spaces or underscores.
 
-* Note down any places you got stuck, problems or errors you ran into while
-doing this setup process. Add your notes as a `notes.md` file to
-`interactive-bots/links/<username>/`.
+* Take note of any places you got stuck and problems or errors you ran into while
+doing this setup process. Add your notes as a `notes.md` file in
+`interactive-bots/followup/<username>/`.
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: In section B change has completely no sense

File: tasks/interactive-bots.md [hunk 23]
```diff
@@ -238,44 +237,44 @@ and Task Type B.
 
 Let *feature* be the feature mentioned in the task that brought you here.
 
-* Make sure you understand what is expected of the *feature* - what messages
-should the bot react to, what should it do with the message content and where
-should it post any output. Should you have any doubts, discuss them in the Zulip
+* Make sure you understand what is expected of the *feature* i.e. what messages
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: dash was correct

File: tasks/interactive-bots.md [hunk 24]
```diff
@@ -238,44 +237,44 @@ and Task Type B.
...
+the bot should react to, what it should do with the message content, and where
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: revert.

File: tasks/interactive-bots.md [hunk 25]
```diff
@@ -238,44 +237,44 @@ and Task Type B.
...
+it should post any output. Should you have any doubts, discuss them in the Zulip
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: revert.

File: tasks/interactive-bots.md [hunk 26]
```diff
@@ -238,44 +237,44 @@ and Task Type B.
...
 `GCI tasks` stream.
 
 * Create the interactive bot at `~/zulip/contrib_bots/lib/<feature>` as a
-`<feature>.py` file. Make sure it fulfills the requirements.
+`<feature>.py` file. Make sure it fulfills the project requirements.
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: revert.

File: tasks/interactive-bots.md [hunk 27]
```diff
@@ -238,44 +237,44 @@ and Task Type B.
...
 
 * Test your bot manually, check that:
 
   * The bot is reacting to the appropriate messages.
   * The bot uses the content from the messages properly.
   * The bot posts the expected output.
 
-* Create a commit with your bot, with commit message `interactive bots: Create
+* Create a commit with your bot, with the commit message `interactive bots: Create
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: revert.

File: tasks/interactive-bots.md [hunk 28]
```diff
@@ -238,44 +237,44 @@ and Task Type B.
...
 *feature* bot.`.
 
-* Create a pull request in the `zulip/zulip` repository, with title
+* Create a pull request in the `zulip/zulip` repository, with the title
 `interactive bots: Create *feature* bot.`. Link to your GCI task in comment section.
 
-*Completion criteria:* Mentors will check if the new bot fulfills the feature
+*Completion criteria:* Your project mentors will check if the new bot fulfills the feature
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: revert.

File: tasks/interactive-bots.md [hunk 29]
```diff
@@ -238,44 +237,44 @@ and Task Type B.
...
 
 ### Task Type D: Create your own bot
 
 We recommend claiming this task after completing Task Type A and Task Type B.
 
 * Come up with an idea for your own interactive bot that would improve Zulip.
-Make sure to make clear what would be expected of the bot -  what messages
-should it react to, what should it do with the message content and where should
-it post any output.
+Make sure to make the bot's function clear -  what messages
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: revert.

File: tasks/interactive-bots.md [hunk 30]
```diff
@@ -238,44 +237,44 @@ and Task Type B.
...
+it should react to, what it should do with the message content and where it should
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: revert.

File: tasks/interactive-bots.md [hunk 31]
```diff
@@ -238,44 +237,44 @@ and Task Type B.
...
+post any output.
 
 * Present your idea for an interactive bot on the `interactive bots` topic on
 the `GCI tasks` stream. Consult with mentors and make sure your idea is approved
  before you start coding.
 
 * Create your interactive bot at `~/zulip/contrib_bots/lib/<bot_name>` as a
-`<bot_name>.py` file. Make sure it fulfills the requirements.
+`<bot_name>.py` file. Make sure it fulfills the feature requirements.
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: revert.

File: tasks/interactive-bots.md [hunk 32]
```diff
@@ -284,14 +283,14 @@ the `GCI tasks` stream. Consult with mentors and make sure your idea is approved
...
-* Create a commit with your bot, with commit message `interactive bots: Create
+* Create a commit with your bot, with the commit message `interactive bots: Create
 <bot_name> bot.`.
 
-* Create a pull request in the `zulip/zulip` repository, with title
+* Create a pull request in the `zulip/zulip` repository, with the title
 `interactive bots: Create <bot_name> bot by <username>.`. Link to your GCI task in comment section.
 
-*Completion criteria:* Mentors will check if the new bot fulfills the feature
+*Completion criteria:* Your project mentors will check if the new bot fulfills the feature
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: revert.

**Comment by TomaszKolek** [comment]
@HogabHerzl Most of your changes are not needed I think - please clean up your PR according to my comments and I will take a look at it again.

File: tasks/interactive-bots.md (hunk 4 above)
Comment (HogabHerzl) [inline tasks/interactive-bots.md]: How about "the interactive bots are python files in the `contrib-bots/lib` that define a bot's specific behavior"?

Comment (HogabHerzl) [inline tasks/interactive-bots.md]: * add the sentence "The `contrib-bots/run.py` file defines common behaviors for the interactive bots that react to messages."

File: tasks/interactive-bots.md (hunk 9 above)
Comment (HogabHerzl) [inline tasks/interactive-bots.md]: I think "include" works better here

File: tasks/interactive-bots.md (hunk 16 above)
Comment (HogabHerzl) [inline tasks/interactive-bots.md]: Perhaps "mets these expectations"?

File: tasks/interactive-bots.md (hunk 14 above)
Comment (HogabHerzl) [inline tasks/interactive-bots.md]: I believe "react" and "post" are correct

File: tasks/interactive-bots.md (hunk 8 above)
Comment (HogabHerzl) [inline tasks/interactive-bots.md]: True

File: tasks/interactive-bots.md (hunk 11 above)
Comment (HogabHerzl) [inline tasks/interactive-bots.md]: "in" works better

File: tasks/interactive-bots.md (hunk 16 above)
Comment (HogabHerzl) [inline tasks/interactive-bots.md]: *meets

**Comment by umairwaheed** [comment]
@HogabHerzl, can you please submit the PR with just one commit? I think you should limit the length of lines to 80 characters.

**Comment by HogabHerzl** [comment]
I'd be happy to, but could you please tell me how to do that, though?

**Comment by umairwaheed** [comment]
@HogabHerzl, just word wrap the lines. Currently, you create a one big line. To wrap it, write your long line on multiple lines. You can ask on Zulip chat.

**Comment by umairwaheed** [comment]
To squash your two commits into one, use `git rebase`.

**Comment by HogabHerzl** [comment]
That solves the wrapping problem mostly, I think, but I'm still not sure how to merge the commits

**Comment by umairwaheed** [comment]
@HogabHerzl, can you squash to just one commit? Thanks.

**Comment by HogabHerzl** [comment]
I suppose I have to use git in the command line in order to do this, if so, I would appreciate you telling me exactly how because I have never used git before, but if there is another way to squash the PR I would be happy to do that as well

**Comment by trueskawka** [comment]
Hi @HogabHerzl!

Here are some helpful docs on git:

- [Git guide](http://zulip.readthedocs.io/en/latest/git-guide.html)
- [working copies](https://github.com/zulip/zulip-gci/blob/master/docs/working-copies.md)
- [fixing commits](https://github.com/zulip/zulip-gci/blob/master/docs/fixing-commits.md)

Please read those and try to clean up your commit history. 

Thanks!

File: tasks/interactive-bots.md [hunk 33]
```diff
@@ -17,38 +17,38 @@
...
 Zulip [contrib-bots](https://github.com/zulip/zulip/tree/master/contrib_bots/lib)
-is a boilerplate for creating interactive bots that react to messages sent
-by users.
+ -is a boilerplate for creating interactive bots that react to messages
+ sent by users.
 
-The interactive bots live in the `contrib-bots/lib` as python files ('.py') that define
-their specific behavior. The `contrib-bots/run.py` file defines common behaviors
-for the interactive bots that react to messages.
+The interactive bots are python files in the contrib-bots/lib that define a
```
Comment (trueskawka) [inline tasks/interactive-bots.md]: This is not right - using \` around `contrib-bots/lib` was intentional.

File: tasks/interactive-bots.md [hunk 34]
```diff
@@ -17,38 +17,38 @@
...
+bot's specific behavior. The `contrib-bots/run.py` file defines common 
+behaviors for the interactive bots that react to messages.
 
-The following tasks will introduce you to using interactive bots and creating
-simple new bots that react to messages. This group of tasks has a high creative
-potential, as you can create your own bots that react to specific messages and
-can be integrated with various APIs.
+The following tasks will introduce you to using interactive bots and 
+creating new simple bots that react to messages. The skills taught in this 
+task can be used to create your own bots that can react to specific
```
Comment (trueskawka) [inline tasks/interactive-bots.md]: It's a group of tasks, not one task.

File: tasks/interactive-bots.md [hunk 35]
```diff
@@ -17,38 +17,38 @@
...
+The interactive bots are python files in the contrib-bots/lib that define a 
+bot's specific behavior. The `contrib-bots/run.py` file defines common 
+behaviors for the interactive bots that react to messages.
 
-The following tasks will introduce you to using interactive bots and creating
-simple new bots that react to messages. This group of tasks has a high creative
-potential, as you can create your own bots that react to specific messages and
-can be integrated with various APIs.
+The following tasks will introduce you to using interactive bots and 
+creating new simple bots that react to messages. The skills taught in this
```
Comment (trueskawka) [inline tasks/interactive-bots.md]: I'm not sure how rephrasing here improves the readability of the doc? The phrasing about creativity was intentional.

File: tasks/interactive-bots.md [hunk 36]
```diff
@@ -78,40 +79,41 @@ Follow the tutorial below to test run the followup bot on your Zulip instance
...
- run the bot and the output with the bot description
+ - your terminal window with the bot running, including the command 
+ you used to run the bot and the output with the bot description
  - messages sent to the bot in the `devel` and `social` streams
  - messages sent by the bot in the followup stream
  - any other screenshots you find relevant
 
-Add the screenshots to `interactive-bots/followup/<username>/`. Make sure your
-filenames include no white spaces or underscores, use dashes (`-`) as connectors.
+Add the screenshots to `interactive-bots/followup/<username>/`. filenames
```
Comment (trueskawka) [inline tasks/interactive-bots.md]: Why did you cut the beginning of the sentence here?

File: tasks/interactive-bots.md [hunk 37]
```diff
@@ -78,40 +79,41 @@ Follow the tutorial below to test run the followup bot on your Zulip instance
...
  - any other screenshots you find relevant
 
-Add the screenshots to `interactive-bots/followup/<username>/`. Make sure your
-filenames include no white spaces or underscores, use dashes (`-`) as connectors.
+Add the screenshots to `interactive-bots/followup/<username>/`. filenames 
+include no white spaces or underscores, use dashes (`-`) as connectors.
 
-* Note down any places you got stuck, problems or errors you ran into while
-doing this setup process. Add your notes as a `notes.md` file to
+* Take note of any places you got stuck, and problems or errors you ran
```
Comment (trueskawka) [inline tasks/interactive-bots.md]: I don't think `and` is needed here.

File: tasks/interactive-bots.md [hunk 38]
```diff
@@ -166,7 +170,7 @@ messages to users.
   return is_link
   ```
 
-  The bot now responds to `@link` and `@resource`.
+  The bot can now detect `@link` and `@resource` in a message.
```
Comment (trueskawka) [inline tasks/interactive-bots.md]: It responds, not detects - I don't think this is right.

File: tasks/interactive-bots.md [hunk 39]
```diff
@@ -207,29 +211,31 @@ messages to users.
...
  - any other screenshots you find relevant
 
- Add the screenshots to `interactive-bots/links/<username>/`. Make sure your
- filenames include no white spaces or underscores, use dashes (`-`) as connectors.
+Add the screenshots to `interactive-bots/links/<username>/`. filenames 
+include no white spaces or underscores, use dashes (`-`) as connectors.
 
-* Note down any places you got stuck, problems or errors you ran into while
-doing this setup process. Add your notes as a `notes.md` file to
+* Take note of any places you got stuck and problems or errors you ran
```
Comment (trueskawka) [inline tasks/interactive-bots.md]: Not sure if `and` is needed here.

File: tasks/interactive-bots.md [hunk 40]
```diff
@@ -207,29 +211,31 @@ messages to users.
...
-* Note down any places you got stuck, problems or errors you ran into while
-doing this setup process. Add your notes as a `notes.md` file to
+* Take note of any places you got stuck and problems or errors you ran 
+into while doing this setup process. Add your notes as a `notes.md` file in
 `interactive-bots/links/<username>/`.
 
-* Create a commit with the screenshots and notes, with commit message
-`interactive bots: Run the links bot for <username>.`.
+* Create a commit with the screenshots and notes, with the commit message
+`interactive bots: Run the followup bot for <username>.`.
```
Comment (trueskawka) [inline tasks/interactive-bots.md]: This is wrong - the task is about the `links` bot.

File: tasks/interactive-bots.md [hunk 41]
```diff
@@ -207,29 +211,31 @@ messages to users.
...
 
-* Create a pull request in the `zulip/zulip-gci` repository, with title
-`interactive bots: Run the links bot for <username>`. Link to your GCI task in comment section.
+* Create a pull request in the `zulip/zulip-gci` repository, with the title
+`interactive bots: Run the followup bot for <username>`. Include a link to
```
Comment (trueskawka) [inline tasks/interactive-bots.md]: `links` bot

File: tasks/interactive-bots.md [hunk 42]
```diff
@@ -207,29 +211,31 @@ messages to users.
...
+`interactive bots: Run the followup bot for <username>.`.
 
-* Create a pull request in the `zulip/zulip-gci` repository, with title
-`interactive bots: Run the links bot for <username>`. Link to your GCI task in comment section.
+* Create a pull request in the `zulip/zulip-gci` repository, with the title
+`interactive bots: Run the followup bot for <username>`. Include a link to 
+your GCI task in comment section.
 
-*Completion criteria:* Mentors will check that the links bot was properly set up.
+*Completion criteria:* Mentors will check that the followup bot was properly
```
Comment (trueskawka) [inline tasks/interactive-bots.md]: `links` bot

File: tasks/interactive-bots.md [hunk 43]
```diff
@@ -239,8 +245,9 @@ and Task Type B.
 Let *feature* be the feature mentioned in the task that brought you here.
 
 * Make sure you understand what is expected of the *feature* - what messages
-should the bot react to, what should it do with the message content and where
-should it post any output. Should you have any doubts, discuss them in the Zulip
+the bot should react to, what it should do with the message content, and 
+where it should post any output. Should you have any doubts, discuss them in 
+the Zulip
 `GCI tasks` stream.
```
Comment (trueskawka) [inline tasks/interactive-bots.md]: This should be on the previous line now.

**Comment by trueskawka** [comment]
I also reviewed the proposed changes and commented on them - please address my comments before you resubmit the task. You should also rebase your feature branch, as the doc has been changed with other PRs.

Thanks!

**Comment by timabbott** [comment]
@HogabHerzl are you planning to finish this?  It'd be great to improve the instructions, since we'll likely use a version of them next year as well.

**Comment by YagoGG** [comment]
Closing this due to inactivity.

@HogabHerzl, feel free to ping us if you're still interested in working on this. Thanks!
